### PR TITLE
Add CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Koncord**
Here is the first version of CONTRIBUTING manual. Since I'm new here, I wrote only common and obvious things.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+* Limit the first line to 72 characters
+
+### Code style
+
+To keep the source consistent and easy to merge, we are using ClangFormat to format code.
+You can find ``.clang-format`` config in the root.
+
+## Pull Request Process
+
+To make approve process easier, pull request, if applicable, should follow this:
```
Comment (Yakvi) [inline CONTRIBUTING.md]: I'd rephrase: 
```suggestion
To make the approval process easier and faster, your pull request should include the following (if applicable):
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+* Reasoning of changes
+* Summary of changes
+
+There are also some tips:
+
+* If you are working on different features, split the job to branches
+* Prefer to rebase over merge when pulling changes from master
+* Avoid mixing tasks in a single Pull Request
+* Give understandable names for commits
+* Be sure that all your change is clear to understand, add have comments for complicated code
```
Comment (Yakvi) [inline CONTRIBUTING.md]:
```suggestion
* Be sure that all your changes are clear to understand.
* Add comments for non-trivial code
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+When contributing to this repository, please check [Issues](https://github.com/storm-devs/storm-engine/issues) and
+[Projects](https://github.com/storm-devs/storm-engine/projects),
+but you will probably start a [discussion](https://github.com/storm-devs/storm-engine/discussions) first.
+
+## Styleguides
+
+### Git commit messages
```
Comment (Yakvi) [inline CONTRIBUTING.md]: capitalization
```suggestion
### Git Commit Messages
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+
+* Use the present tense and imperative mood
+* Limit the first line to 72 characters
+
+### Code style
```
Comment (Yakvi) [inline CONTRIBUTING.md]:
```suggestion
### Code Style
```

**Review by Yakvi — CHANGES_REQUESTED** [review]
Added some grammar suggestions.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+# Contributing
+
+When contributing to this repository, please check [Issues](https://github.com/storm-devs/storm-engine/issues) and
+[Projects](https://github.com/storm-devs/storm-engine/projects),
+but you will probably start a [discussion](https://github.com/storm-devs/storm-engine/discussions) first.
```
Comment (np-c0mp1ete) [inline CONTRIBUTING.md]: I'd also mention the discord channel here:

> When contributing to this repository, please check [Issues](https://github.com/storm-devs/storm-engine/issues) and
[Projects](https://github.com/storm-devs/storm-engine/projects). Also, it's a good idea to drop a message on our [Discord channel](https://discord.gg/jmwETPGFRe) or start a [discussion](https://github.com/storm-devs/storm-engine/discussions) first to let others know what you're planning to work on.

**Review by np-c0mp1ete — COMMENTED** [review]
Should we also update the [Contributing](https://github.com/storm-devs/storm-engine#contributing) section of the Readme or maybe remove it completely?

File: CONTRIBUTING.md (hunk 5 above)
Comment (espkk) [reply CONTRIBUTING.md]: I think we should leave a link to contributing.md in that section

**Review by np-c0mp1ete — APPROVED** [review]
